server/public/shared/mlog/mlog.go

```diff
@@ -98,32 +98,25 @@ func (lc LoggerConfiguration) toTargetCfg() map[string]logrcfg.TargetCfg {
 // will be used to generate a string representation.
 var Any = logr.Any
 
-// Int64 constructs a field containing a key and Int64 value.
-var Int64 = logr.Int64
-
-// Int32 constructs a field containing a key and Int32 value.
-var Int32 = logr.Int32
-
-// Int constructs a field containing a key and Int value.
-var Int = logr.Int
-
-// Uint64 constructs a field containing a key and Uint64 value.
-var Uint64 = logr.Uint64
-
-// Uint32 constructs a field containing a key and Uint32 value.
-var Uint32 = logr.Uint32
-
-// Uint constructs a field containing a key and Uint value.
-var Uint = logr.Uint
+// Int constructs a field containing a key and int value.
+func Int[T ~int | ~int8 | ~int16 | ~int32 | ~int64](key string, val T) Field {
+	return logr.Int[T](key, val)
+}
```

Comment:
I don't think it's possible to alias a generic function.

Reply:
That's unfortunate.  My understanding is that inlining is also disabled for generic methods.